lib/absinthe/schema/notation.ex

```diff
@@ -1201,15 +1201,19 @@ defmodule Absinthe.Schema.Notation do
       attrs
       |> handle_deprecate
       |> Keyword.put(:identifier, identifier)
-      |> Keyword.put(:name, to_string(identifier))
+      |> Keyword.put_new(:name, to_string(identifier))
       |> put_reference(env)
 
     struct!(Schema.InputValueDefinition, attrs)
   end
 
   def record_arg!(env, identifier, attrs) do
     arg = build_arg(identifier, Keyword.put(attrs, :module, env.module), env)
-    put_attr(env.module, arg)
+    ref = put_attr(env.module, arg)
+
+    [
+      get_desc(ref)
+    ]
```

Comment:
Ah yeah, if we weren't adding the `get_desc` call here, then it wouldn't have ended up on the stack at all.